adal-accounts/src/main/java/com/massivedisaster/adal/account/AccountHelper.java

```diff
@@ -138,52 +138,73 @@ private static void removeAccounts(final OnAccountListener onAccountListener, fi
     }
 
     /**
-     * Verify if the application has account added.
+     * Verify if the application  have a com.clubefashion.services.account and retrieve The account
```

Comment:
@jzeferino you copy the comment with the app package name, because i din't realize that comments that the comments were like this when I gave you the code